popups/scratch-messaging/popup.css

```diff
@@ -6,6 +6,14 @@ button {
   background: #2a2a2a;
   padding: 10px;
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.25);
+  overflow: hidden;
+}
+
+.contents {
+  max-height: 95vh;
+  overflow-y: auto;
+  scrollbar-width: thin;
+  scrollbar-color: grey transparent;
```

Comment:
We need to double check the scrollbar looks the same in Chrome and Firefox

Reply:
LGTM